### DISCUSSION TITLE
training on city scape dataset

### DISCUSSION THREAD

**Discussion by omarfahmy** [facebookresearch/detectron2#2509] [Q&A]
Dear Sir,
i'd like to run detectron 2 on built-in cityscape dataset. I tried to open and run Colab Notebook but i failed to run on cityscape dataset.
Could you help me how to train on cityscape or any other custom dataset?..

Thanks & Best Regards

**Comment by deepaksinghcv** [comment]
Where did it fail and what was the error, while training on cityscapes?

**Reply by omarfahmy** [reply]
Thank you; i have succeeded to train and evaluate on cityscapes; the error was in loading wrong config file....

**Comment by melkamu12** [comment]
![image](https://github.com/user-attachments/assets/d40f2868-66ed-4acf-a8b9-ed558cc58109)
